feature_layers/feature-collection-layer-from-portal/README.metadata.json

```
@@ -0,0 +1,35 @@
+{
+    "category": "Feature layers",
+    "description": "Combine feature tables with different geometries into a single layer.",
```

This looks like it's from another sample 😄 . It should be the same intro sentence as in the common design
```suggestion
    "description": "Create a feature collection layer from a portal item.",
```

---

that is true it was copied from another sample and forgot to update